alert-database/src/main/java/com/synopsys/integration/alert/database/api/DefaultAuditUtility.java

```diff
@@ -134,7 +134,7 @@ public DefaultAuditUtility(AuditEntryRepository auditEntryRepository, AuditNotif
                                                   .concat(componentNotificationIds.stream(), topLevelActionNotificationIds.stream())
                                                   .collect(Collectors.toSet());
         for (Long notificationId : allMessageNotificationIds) {
-            AuditEntryEntity auditEntryEntity = new AuditEntryEntity(jobId, new Date(System.currentTimeMillis()), null, null, null, null);
+            AuditEntryEntity auditEntryEntity = new AuditEntryEntity(jobId, OffsetDateTime.now(), null, null, null, null);
```

Comment:
I think we should be using `DateUtils. createCurrentDateTimestamp()` across Alert when we are getting the current time.